koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -760,35 +771,46 @@ def populate_monthly_cost(self, node_cost, start_date, end_date):
                 Max('usage_end')
             )['usage_end__max']
 
-        LOG.info('Populating Monthly cost from %s to %s.', start_date, end_date)
-
         first_month = start_date.replace(day=1, hour=0, minute=0, second=0, microsecond=0)
 
         with schema_context(self.schema):
             # Calculate monthly cost for every month
             for curr_month in rrule(freq=MONTHLY, until=end_date, dtstart=first_month):
                 first_curr_month, first_next_month = month_date_range_tuple(curr_month)
+                LOG.info('Populating Monthly node cost from %s to %s.',
+                         first_curr_month, first_next_month)
 
                 unique_nodes = OCPUsageLineItemDailySummary.objects.\
                     filter(usage_start__gte=first_curr_month,
                            usage_start__lt=first_next_month,
                            node__isnull=False
```

Comment:
I think we need to add `cluster_id=cluster_id` to this filter.

Reply:
ahhh. That is a great catch! 👍 I will make that change now.

Reply:
https://github.com/project-koku/koku/pull/1685/files#diff-5132f3be415e3032124d987a0aff4939R786